src/app/books/book-details/book-details.component.ts

```diff
@@ -36,9 +36,29 @@ export class BookDetailsComponent implements OnInit {
   }
 
   addToCart() {
-    var status : string = this.cart.sendProduct(this.book);
+    var status : string="Duplicate";
```

Comment:
This add to cart should be replaced by Cart Service function

Reply:
done